docs/index.md

```diff
@@ -103,17 +100,17 @@ In addition to [generic `provider` arguments](https://www.terraform.io/docs/conf
 
 * `insecure` - (Optional) This setting allows the provider to skip TLS verification.
   Useful when targeting installation with self-signed certificates. Not recommended when
-  targeting the Elasticsearch Service (ESS).
+  targeting ESS.
 
 * `insecure` - (Optional) This setting allows the user to set a custom timeout in the
-  individual HTTP request level. Defaults to "1m" but might need to be tweaked if timeouts
+  individual HTTP request level. Defaults to 1 minute, but might need to be tweaked if timeouts
```

Comment:
I think we might want to specify what the default value looks like? I'll leave it up to you to decide though :)

```suggestion
  individual HTTP request level. Defaults to 1 minute (`"1m"`), but might need to be tweaked if timeouts
```

Reply:
Your suggestion looks much clearer. Quotes must be kept?